### ISSUE TITLE
Installation blocked at publishing product infomation

### ISSUE THREAD

**Issue by TENX-S** [jasongin/nvs#209]
![image](https://user-images.githubusercontent.com/40336192/121619795-bd52e480-ca9b-11eb-96d7-2d35e0058bbd.png)

The installation process took over 30 minutes on this page. Is this the expected behavior ?

**Comment by TENX-S** [comment]
And the uninstallation is stuck at the beginning ...
![image](https://user-images.githubusercontent.com/40336192/121620564-21c27380-ca9d-11eb-8fea-ba83b15ed50f.png)

**Comment by jasongin** [comment]
Certainly not expected. To find out what is hanging, I'd need to see the installer log:

```
msiexec /i nvs.msi /l*v install.log
```

Or instead of the MSI you can use the [manual installation steps](https://github.com/jasongin/nvs/blob/master/doc/SETUP.md#manual-setup---command-prompt).

**Comment by Maxim-Mazurok** [comment]
Exactly the same for me

**Comment by Maxim-Mazurok** [comment]
Please, see [install.log](https://github.com/jasongin/nvs/files/7111665/install.log)

TL;DR: a bunch of these:
```
CAQuietExec:  C:\Windows\SysWOW64\%ProgramFiles%\cache\node.exe
CAQuietExec:  Access is denied.
```

**Comment by Maxim-Mazurok** [comment]
1.5.4 installed fine: 
[install.log](https://github.com/jasongin/nvs/files/7111681/install.log)

But getting this:
```
C:\Program Files\cache\node.exe
Access is denied.
```

and this as admin:
```
C:\Windows\system32>nvs
Downloading boostrap node from https://nodejs.org/dist/v10.12.0/node-v10.12.0-win-x64.7z

EPERM: operation not permitted, lstat 'C:\Program Files\Windows Defender Advanced Threat Protection\Classification\Configuration'
Try running again as Administrator.
```

**Comment by Maxim-Mazurok** [comment]
Using `choco install nvs` (1.5.2) results in
```
C:\Users\maxim>nvs
EPERM: operation not permitted, lstat 'C:\Program Files\Windows Defender Advanced Threat Protection\Classification\Configuration'
Try running again as Administrator.
```
when running from both admin and not admin.

**Comment by Maxim-Mazurok** [comment]
I also tried to disable all of these:
- Real-time protection
- Cloud-delivered protection
- Automatic sample submission
- Tamper protection

didn't help.

**Comment by Maxim-Mazurok** [comment]
As per https://github.com/jasongin/nvs/issues/217#issuecomment-913275223 deleting NVS_HOME env var and reinstalling should help.
